specification/schemas/responses/MessageCreated.yaml

```diff
@@ -16,10 +16,7 @@ properties:
         additionalProperties: false
```

Comment:
Just noticed that we are missing the `Location` header on the POST /v1/messages response!